### PR TITLE
feat: final polish of notebook; clean up

### PR DESCRIPTION
**Pull request by telpirion**
This pull request provides the following fixes and updates:

+ Adds component-by-component steps in notebook
+ Removes references to Workbench in notebook
+ Removes TODOs from code
+ Removes unneeded / incorrect logging and print statements

### CODE REVIEW

File: notebook/gen_ai_jss.ipynb [hunk 1]
```diff
@@ -165,29 +159,104 @@
...
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 6,
    "metadata": {
     "id": "2b4ef9b72d43"
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
```
Comment (balajismaniam) [inline notebook/gen_ai_jss.ipynb:195]: Reminder: clear outputs.

File: notebook/gen_ai_jss.ipynb [hunk 2]
```diff
@@ -248,7 +336,7 @@
     "   - [Cloud Vision API](https://console.cloud.google.com/flows/enableapi?apiid=vision.googleapis.com)\n",
```
Comment (balajismaniam) [inline notebook/gen_ai_jss.ipynb:336]: There is a TODO in https://github.com/GoogleCloudPlatform/terraform-genai-doc-summarization/pull/45/files#diff-6fb1d64e4b780f84f096c075eef170028d99b4e41867cb48cd7ff7983d22a70eL238. Please fix that as well. 

Terraform script link: https://github.com/GoogleCloudPlatform/terraform-genai-doc-summarization
Solutions console link: https://pantheon.corp.google.com/products/solutions/details/generative-ai-document-summarization

**Review by balajismaniam — APPROVED** [review]
LGTM. I was able to run the notebook and I deployed the branch in SiC to test. Working as expected.

File: notebook/gen_ai_jss.ipynb (hunk 2 above)
Comment (telpirion) [reply notebook/gen_ai_jss.ipynb:243]: Done.

File: notebook/gen_ai_jss.ipynb (hunk 1 above)
Comment (telpirion) [reply notebook/gen_ai_jss.ipynb]: Done.

**Pull request merged**